lib/adapter/freshheads/BabelLoaderAdapter.ts

```diff
@@ -35,7 +35,9 @@ export default class BabelLoaderAdapter implements Adapter {
         _builderConfig: BuilderConfig,
         next: NextCallback
     ) {
-        this.validateAllRequiredModulesAreInstalled();
+        if (_builderConfig.env === Environment.Dev) {
```

Comment:
de `_`-prefix mag weg hoor! Die is er alleen maar omdat in deze adapter deze variabele niet gebruikt werd, en anders Typescript daar moeilijk over doet. (je kunt 'm tenslotte ook niet weglaten)